builders/tooling/oracles/acurast.md

```diff
@@ -0,0 +1,146 @@
+
+    ```solidity
+    --8<-- 'code/builders/tooling/oracles/acurast/InsecureDummyPriceFeed.sol'
+    ```
+
+
+!!! warning
+    This demo contract has a number of security vulnerabilities and lacks access control mechanisms, making it unsuitable for any real use. It was developed by the Acurast team for demonstration purposes only. 
+
+You will need to deploy your price feed's smart contract on your EVM Appchain prior to continuing to the following steps. After deployment, make a note of the contract address because you'll need to provide it as input to your Acurast price feed script.  
+
+### Building the Acurast Script
+
+The Acurast oracle script is what feeds your onchain oracle with new data. It's the essential link between your price feed accessible on your Tanssi Appchain and the Acurast network. In the Acurast console, you'll upload your script and provide all of the relevant parameters surrounding your price feed's operation, such as frequency, schedule, reward to Acurast processors, and more. You'll need some cACU tokens to pay for the job, which you can obtain from the [faucet](https://faucet.acurast.com/){target=\_blank}.
+
+The [Acurast script for the demo BTC/USD price feed](https://github.com/Acurast/acurast-evm-oracle-sample/blob/main/acurast_scripts/oracle_job.js){target=\_blank} can be used a basis for creating your own script. Remember to update the contract address and RPC URL fields.
+
+??? code "AcurastScript.js"
+
+    ```js
+    --8<-- 'code/builders/tooling/oracles/acurast/AcurastScript.js'
+    ```
+
+
+To configure your job, head to the [Acurast console](https://console.acurast.com/create){target=\_blank}, then take the following steps:
+
+1. Click **Create Jobs** on the left-hand sidebar underneath the **Consumer** heading
+2. Select **Moonbeam** as the chain 
+3. Select **Moonbase** as the environment. Remember that Tanssi's EVM-compatibility is derived from Moonbeam
+4. Select **Price Feeds**
+5. Paste in the code of your job script. You can copy and paste directly from the [script of the sample BTC/USD price feed](https://github.com/Acurast/acurast-evm-oracle-sample/blob/main/acurast_scripts/oracle_job.js){target=\_blank}, just make sure to change the destination contract to one that you deployed on your Appchain and the RPC node to your [Appchain's RPC URL](https://apps.tanssi.network/)
+6. Optionally, you can test your code here. Any error messages will be readable in the browser's console
+
+![Job setup on Acurast console](/images/builders/tooling/oracles/acurast/acurast-4.webp)
+
+Continuing down the same setup page, take the following steps: 
+
+1. Select **Use Public Processors**
+2. Select **Interval**
+3. Specify a **Start time** and **End time**
+4. Specify the **Interval in minutes**
+5. Specify a job duration and max start delay duration
+6. Select **Number of processors to assign**. The more processors you choose, the proportionally higher amount of cACU you'll need
```

Comment:
what is `cACU` (I know it is the token, but I believe it was not well explained in the write up)

Reply:
Also, how do you get `cACU`?

Reply:
Added, also there is the link to the faucet